pkg/workspace/inference/preset_inferences.go

```diff
@@ -352,14 +382,14 @@ func GenerateInferencePodSpec(gpuConfig *sku.GPUConfig, numNodes int) func(*gene
 				if presetParams != nil {
 					if raw, ok := configVolume.Data["inference_config.yaml"]; ok && raw != "" {
 						// First check if user provided explicit value in ConfigMap
-						//if v, ok2 := utils.ParseExplicitMaxModelLen(raw); ok2 {
-						//maxModelLen = v
-						//klog.Infof("[RuntimeContext] workspace=%s using user explicit max-model-len=%d", ctx.Workspace.Name, maxModelLen)
-						//} else {
-						// If no user value, compute planned value
-						maxModelLen = computeMaxModelLen(presetParams, gpuConfig, numNodes)
-						klog.Infof("[RuntimeContext] workspace=%s using computed max-model-len=%d (gpuConfig=%+v, numNodes=%d)", ctx.Workspace.Name, maxModelLen, *gpuConfig, numNodes)
-						//}
+						if v, ok2 := utils.ParseExplicitMaxModelLen(raw); ok2 {
```

Comment:
@bangqipropel needs to comment what the right change should be. This old logic seems to be problematic to me.